Source/WebCore/platform/graphics/skia/SkiaAcceleratedBufferPool.cpp

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2024 Igalia S.L.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public License
+ * along with this library; see the file COPYING.LIB.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#include "config.h"
+#include "SkiaAcceleratedBufferPool.h"
+
+#if USE(COORDINATED_GRAPHICS) && USE(SKIA)
+#include "GLContext.h"
+#include "PlatformDisplay.h"
+#include <skia/gpu/GrBackendSurface.h>
+#include <skia/gpu/ganesh/SkSurfaceGanesh.h>
+#include <skia/gpu/ganesh/gl/GrGLBackendSurface.h>
+#include <skia/gpu/ganesh/gl/GrGLDirectContext.h>
+#include <skia/gpu/gl/GrGLInterface.h>
+#include <skia/gpu/gl/GrGLTypes.h>
+
+namespace WebCore {
+
+SkiaAcceleratedBufferPool::SkiaAcceleratedBufferPool()
```

Comment:
Since this is an existing issue and unrelated to threaded rendering, I would do this in a separate patch.

Reply:
Fixed, created PR #24690.